lib/DDG/Spice/Riffsy.pm

```diff
@@ -10,10 +10,13 @@ spice proxy_cache_valid => "200 1d";
 spice to => 'http://api1.riffsy.com/v1/search?tag=$1&key={{ENV{DDG_SPICE_RIFFSY_APIKEY}}}&safesearch=strict&searchtype=ddg&limit=50';
 spice wrap_jsonp_callback => 1;
 
-triggers query_lc => qr/^\X$|^(\X+|[\w\s]+?) gifs?$/;
+triggers query_lc => qr/^(?<match>\X)$|^(\X+|[\w\s]+?) gifs?$/;
 
 handle query_lc => sub {
     return if !$_ || $_ =~ /^[a-zA-Z0-9]$/;
+    my @emojis = map { chomp $_; $_ } share('emoji_sources.txt')->slurp;
```

Comment:
Let's move this outside the `handle` so we only build this regex once and store it in memory for the `handle` function to use when needed